src/assets/icons/Imgheart.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="23" height="23" viewBox="0 0 23 23" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Do you already use this icon. If yes, could you please import it according to this https://mobile.twitter.com/marksshenouda/status/1491510779685789697

Reply:
We use it in _generalSettings.scss as background. Is it good or no?

Reply:
Okay, that's fine for this case